static/js/components/EducationTab.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Button from 'react-mdl/lib/Button';
+import ProfileTab from "../util/ProfileTab";
+import { boundEducationSelectField,
+  boundEducationMonthField, boundEducationYearField,
+  saveAndContinue } from '../util/profile_edit';
+import {
+  toggleEducationLevel,
+  openEducationForm,
+  closeEducationForm
+} from '../actions';
+import { Grid, Cell, Switch, Dialog, DialogTitle, DialogContent, DialogActions, FABButton, Icon } from 'react-mdl';
+
+class EducationTab extends ProfileTab {
+  constructor(props) {
+    super(props);
+    this.boundEducationMonthField = boundEducationMonthField.bind(this);
+    this.boundEducationYearField = boundEducationYearField.bind(this);
+    this.educationLevelOptions = [
+      { value: 'hs', label: "High school", enabled: true },
```

Comment:
Can we define each of the values here as constants in `constants.js` since we use these in the reducers too?